Docs: Slightly reconstruct the "Smooth upgrade mount pod" document (#1151)
juicedata/juicefs-csi-driver: docs/en/guide/custom-image.md

* `/usr/local/bin/juicefs`: JuiceFS Community Edition client
* `/usr/bin/juicefs`: JuiceFS Enterprise Edition client

> To avoid misuse and reduce image size, from CSI Driver 0.19.0, separated image is provided for CE/EE, you can find the latest mount pod image in [Docker Hub](https://hub.docker.com/r/juicedata/mount/tags), the image tag looks like:

Reviewer: ```suggestion
To avoid misuse and reduce image size, from CSI Driver 0.19.0, separated images are provided for CE/EE. You can find the latest Mount Pod image in [Docker Hub](https://hub.docker.com/r/juicedata/mount/tags). The image tag looks like:
```

:::tip

> If you need to move the Mount Pod image from Docker Hub to other container registry, please refer to [documentation](../administration/offline.md#copy-images).

Reviewer: ```suggestion
If you need to move the Mount Pod image from Docker Hub to another container registry, please refer to the [documentation](../administration/offline.md#copy-images).
```

## Overwrite mount pod image {#overwrite-mount-pod-image}

:::tip

> The JuiceFS CSI Driver supports [smooth upgrade of mount pod](../administration/upgrade-juicefs-client.md#smooth-upgrade) starting from version 0.25.0. It is recommended to use this method to upgrade mount pod first.

Reviewer: ```suggestion
The JuiceFS CSI Driver supports [smooth upgrade of Mount Pods](../administration/upgrade-juicefs-client.md#smooth-upgrade) starting from version 0.25.0. It is recommended to use this method to upgrade Mount Pods first.
```
